packages/engine/README.md

```diff
@@ -142,6 +143,16 @@ Then, run the CLI using:
 Where CLI args are described below in the [Usage](#usage) section, an example of a run command during development would be:
 * `cargo run --bin cli -- <CLI ARGS> -p  "<PATH TO HASH PROJECT DIR>" single-run --num-steps <NUM-STEPS>`
 
+### Contributing 
```

Comment:
This would perhaps be best as its own README that we link to here, can follow-up on slack

Reply:
We can redirect this PR into the `engine/contributions` branch while we wait on #107 to merge